CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ endif()
 # Adjust compiler settings
 if(MSVC)
     add_compile_options(/MP)
+    add_compile_options(/bigobj)
```

Comment:
Is there an advantage to including this setting in non-monolithic Windows builds of MaterialX, or should we restrict this change using MATERIALX_BUILD_MONOLITHIC?

Reply:
/bigobj just increases the number of sections that can be linked, and shouldn't in general by measurable in any way otherwise. The only "downside" of setting it on always is that if someone is compiling with vs2019+ they won't be able to give the library to someone linking with vs2005 or earlier. The advantage presumably for always setting it is simpler build logic.

Reply:
To be on the safer side, I think it's better to enable it for MATERIALX_BUILD_MONOLITHIC. I'll update the PR.